### DISCUSSION TITLE
Need to populate stencil paper shapes dynamically with text  next to it.

### DISCUSSION THREAD

**Discussion by naresh-FD** [clientIO/joint#2021] [Q&A]
![image](https://user-images.githubusercontent.com/50324374/216602743-0b31f0f4-abdc-47a6-a634-d0bd3d563aac.png)

need to align like in the above image.

Thanks in Advance.

**Comment by jamesgeorgewilliams** [comment]
Duplicate of #1965

**Reply by kumilingus** [reply]
If you need the stencil label to be on the left, please see the modified [example](https://codepen.io/jointjs/pen/KKBYXrg).
